addons/l10n_ec/data/account_fiscal_position_template.xml

```diff
@@ -16,12 +16,12 @@
         <field name="vat_required" eval="True"/>
     </record>
     <record id="fp_foreign_sale_vat12_goods" model="account.fiscal.position.tax.template">
-        <field name="tax_src_id" ref="l10n_ec.tax_vat_411_goods"/>
+        <field name="tax_src_id" ref="l10n_ec.tax_vat_15_411_goods"/>
```

Comment:
Why do not we just add a new line?  Because now xml_id of the line and the tax do not match, ...